src/app/components/SocialLinks/index.tsx

```diff
@@ -62,7 +67,13 @@ const SocialLinkImage = ({ imageUrl }: { imageUrl: string }) => {
   );
 };
 
-const SocialLink = ({ summary }: { summary: Summary }) => {
+const SocialLink = ({
+  summary,
+}: {
+  summary: Summary;
+  eventTrackingData?: EventTrackingData;
+  clickTrackerHandler?: ReturnType<typeof useClickTrackerHandler>;
```

Comment:
`eventTrackingData` and `clickTrackerHandler` aren't being used in here?

I don't think `eventTrackingData` is needed, but `clickTrackerHandler` will need to be used on the anchor tag here to register a click event.